### ISSUE TITLE
Compatibility warning on graceful-fs indirect dependency

### ISSUE THREAD

**Issue by ghusse** [taptapship/wiredep#232]
When installing wiredep, npm gives me this warning:

> npm WARN deprecated graceful-fs@3.0.8: graceful-fs version 3 and before will fail on newer node releases. Please update to graceful-fs@^4.0.0 as soon as possible.

Graceful fs seems to be an indirect dependency of wiredep 3.0.0

```
// npm install output
`-- wiredep@3.0.0 
  +-- bower-config@0.5.2 
  | +-- graceful-fs@2.0.3 
```

I checked the last version of `bower-config`, and its dependency on `graceful-fs` had been updated to 4.0.0

Could you update your dependency as well?

**Comment by ghusse** [comment]
By the way, I also have this message regarding the lodash dependency (which is less problematic):

> npm WARN deprecated lodash@2.4.2: lodash@<3.0.0 is no longer maintained. Upgrade to lodash@^4.0.0

**Comment by eddiemonge** [comment]
Fixed in v3.0.1
